### ISSUE TITLE
ReactTestUtils Warning

### ISSUE THREAD

**Issue by samblake** [Microsoft/TypeScript-React-Starter#25]
When running tests as described in the readme I get the following warning.

`ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.`

To fix this I added a dependency on `react-test-renderer` as [required for Enzyme with React >= 15.5](https://github.com/airbnb/enzyme/blob/75d1390147e1028301274be659dbfa7105550015/README.md#installation).

**Comment by DanielRosenwasser** [comment]
Hey @wmonk, would it make more sense to add this to the https://github.com/wmonk/create-react-app-typescript repo?

**Comment by wmonk** [comment]
We just upgraded to cra@1.x.x so might have been fixed in there. Nobody else has opened this issue there, but if it persists feel free to!

**Comment by DanielRosenwasser** [comment]
@samblake can you try again and see if the issue is still present?

**Comment by samblake** [comment]
I tried again, seems to be fine now.

**Comment by samblake** [comment]
Sorry, got trigger happy, the problem is still present.

```
 PASS  src/components/Hello.test.tsx
  ● Console

    console.error node_modules/fbjs/lib/warning.js:36
      Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.
    console.error node_modules/fbjs/lib/warning.js:36
      Warning: Shallow renderer has been moved to react-test-renderer/shallow. Update references to remove this warning.
````
